_posts/2021-11-24-sprint-report-2.md

```diff
@@ -0,0 +1,62 @@
+---
+author: Tyler Weaver
+comments: false
+date: 2021-11-30 17:00:00+07:00
+layout: post
+slug: sprint-report-2
+title: MoveIt Sprint Report
+description: Nov 5 - Nov 25
+media_type: image
+media_link: /assets/images/blog_posts/2021_11_24_sprint_update.png
+categories:
+- MoveIt
+- ROS
+---
+
+We are on our second MoveIt sprint using ZenHub for planning and would like to share with you what the PickNik team is doing to improve MoveIt.  From a high level the major efforts of this sprint were:
+
+- [Abishalin](https://github.com/Abishalini) has taken over forward porting work from MoveIt on ROS 1 and finished a PR for testing of TrajectoryMonitor.
+- [AndyZe](https://github.com/AndyZe) is working to get [sjahr](https://github.com/sjahr)'s Hybrid Planning feature finished.
+- [DLu](https://github.com/DLu) is working on improving robot_description handling.
+- [henningkayser](https://github.com/henningkayser) updated documentation, did many reviews, and lead the team.
+- [JafarAbdi](https://github.com/JafarAbdi) cut a Foxy release to get bug fixes out into debains.
+- [schornakj](https://github.com/schornakj) added functionality to MoveItControllerManager to let it manage more than just trajectory execution controllers.
+- [stephanie-eng](https://github.com/stephanie-eng) added descriptions and default values to parameters for moveit_servo.
+- [tylerjw](https://github.com/tylerjw) continued the project of making the documentation more correct and user-friendly.
+- [vatanaksoytezer](https://github.com/vatanaksoytezer) added a Mergify bot to CI and helped with the tutorial website update.
+
+### MoveIt 2
```

Comment:
We got https://github.com/ros-planning/moveit_task_constructor/pull/170 merged, you may want to add it here, too